web3/main.py

```diff
@@ -323,6 +331,35 @@ def solidityKeccak(cls, abi_types: List[TypeStr], values: List[Any]) -> bytes:
         )))
         return cls.keccak(hexstr=hex_string)
 
+    def attach_module(
+        self,
+        module_name: str,
+        module: Union[Type[Module], Sequence[Any]]
+    ) -> None:
+        """
+        Attach a module to the `Web3` instance. Modules should inherit from the `web3.module.Module`
+        class.
+
+        Attaching a simple module:
```

Comment:
I added this before the docs. Perhaps this is a bit redundant? Thoughts?

Reply:
I'd vote to only maintain these docs in one place so it's less likely that they get out of sync, but if you feel strongly about leaving them I'm fine with that too.

Reply:
Don't feel strongly. I'm not even sure how many users really use the `__doc__` as an instruction. We can keep the examples in the actual documentation then 👌 . I think if we start adding this to every method, our files just get way too lengthy. I will take this out 👍